### PR TITLE
Issue8

### PR DESCRIPTION
**Pull request by kvsari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,17 +36,21 @@ struct MyCheesyStruct {
...
 ```rust
-pub trait MyCheesyStructGetters {
-    fn get_x(&self) -> i64;
-    fn get_y(&self) -> i64;
+impl MyCheesyStruct {
+    fn get_x(&self) -> i64 {
+        &self.x
+    }
+
+    fn get_y(&self) -> i64 {
```
Comment (LukasKalbertodt) [inline README.md:46]: I guess you forgot to change the function names here?

Comment (kvsari) [reply README.md:46]: It's fixed. Try 0.0.7.

Comment (LukasKalbertodt) [reply README.md:46]: I meant you forgot to change them in the README :) It's still wrong in the README on `master`

Comment (kvsari) [reply README.md:46]: I've fixed it.

**Pull request merged**